modules/app/src/main/java/org/locationtech/jtstest/function/SpatialIndexFunctions.java

```diff
@@ -117,6 +117,14 @@ public static Geometry strTreeNN(Geometry geoms, Geometry geom)
     return (Geometry) result;
   }
 
+  public static Geometry[] strTreeNNk(Geometry geoms, Geometry geom)
```

Comment:
You can add more parameters.  See this example:  

https://github.com/locationtech/jts/blob/master/modules/app/src/main/java/org/locationtech/jtstest/function/CreateShapeFunctions.java#L73